object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -324,11 +342,12 @@ private boolean isTransferOwnershipAtReturn(ControlFlowGraph cfg) {
     return false;
   }
 
-  private void handleAssignment(AssignmentNode node, Set<LocalVarWithTree> newDefs) {
+  private Set<ImmutableSet<LocalVarWithTree>> handleAssignment(
```

Comment:
Can we just return `void` here now?

Reply:
https://github.com/kelloggm/object-construction-checker/pull/242/commits/73a5d0d6f607756067b47b9b65bd846ff1a9c71d